api/tests/Api/Activities/CreateActivityTest.php

```diff
@@ -509,6 +574,7 @@ public function getExampleWritePayload($attributes = [], $except = []) {
             Activity::class,
             Post::class,
             array_merge([
+                'copyActivitySource' => null,
```

Comment:
would it also work without this additional property?

Reply:
nop; `getExampleReadPayload()` does set it with a defaultvalue